app/pages/sections/order-items/catalogue-solutions/order-item/contextCreator.js

```diff
@@ -15,13 +15,17 @@ export const getContext = ({
   selectedPrice,
   selectedRecipients,
   errorMap,
+  catalogueItemAlreadyExist,
```

Comment:
I think either `catalogueItemExists` or `isExistingCatalogueItem` might be slightly better name.

Reply:
changed to catalogueItemExists